job-runner/api/api.go

```diff
@@ -184,7 +185,7 @@ func (api *API) newJobChainHandler(c echo.Context) error {
 	}
 
 	// Set the location in the response header to point to this server.
-	c.Response().Header().Set("Location", chainLocation(jc.RequestId, os.Hostname))
+	c.Response().Header().Set("Location", api.baseURL)
```

Comment:
Why this change? Location should be a URI, i.e. a full path to the returned resource (the job chain, in this case), not just the base URL.

Reply:
Having reviewed rest of code, I think I see why this was changed: so RM can set column `jr_url` = to this Location. I think it's better to keep Location true to purpose (full URL of resource), and in RM use https://golang.org/pkg/net/url/#Parse to parse out the parts needed.

Reply:
https://play.golang.org/p/TYR4RcTUzgV might do the trick